corehq/apps/sms/forms.py

```diff
@@ -296,6 +303,15 @@ def section_registration(self):
                 ),
                 data_bind="visible: showRegistrationOptions",
             ),
+            FieldWithHelpBubble(
+                "sms_mobile_worker_registration_enabled",
+                help_bubble_text=_("When this option is enabled, a person "
+                    "can send an SMS into the system saying 'join "
+                    "[project] worker [username]' (where [project] is your "
+                    " project space and [username] is an optional username)"
+                    ", and the system will add them as a mobile worker."),
+                data_bind="value: sms_mobile_worker_registration_enabled",
```

Comment:
I don't think so - we don't do any show/hide based off of it so it's really not needed.